src/modules/infobox/components/styling/FeatureStyleEdit.vue

```diff
@@ -150,16 +160,60 @@ function onDelete() {
             <label class="form-label" for="drawing-style-feature-description">
                 {{ $t('modify_description') }}
             </label>
-            <textarea
-                id="drawing-style-feature-description"
-                v-model="description"
-                :readonly="readOnly"
-                data-cy="drawing-style-feature-description"
-                class="feature-description form-control"
-                :class="{
-                    'form-control-plaintext': readOnly,
-                }"
-            ></textarea>
+            <div>
+                <div class="d-flex justify-content-end align-items-center">
+                    <DrawingStylePopoverButton
+                        ref="mediaLinkPopover"
+                        data-cy="drawing-style-link-button"
+                        button-class-override="rounded-0 rounded-top-2 rounded-end-0 btn btn-sm btn-light d-flex align-items-center"
+                        icon="fa-link"
+                    >
+                        <DrawingStyleMediaLink
+                            media-type="link"
+                            :url-description="$t('url_link')"
+                            :extra-url-description="$t('text_to_display')"
+                            @generated-media-link="onAddMediaLink"
+                        >
+                        </DrawingStyleMediaLink>
+                    </DrawingStylePopoverButton>
```

Comment:
Might make sense to have an Array of JSON Object that describe what type of media it is, what i18n key to use, and some extra stuff relevant to only one type if necesary, then loop over this to create this templating (meaning only one declaration of the component in the template with a v-for)